### ISSUE TITLE
scrollOverflow causes react-fullpage to throw Maximum call stack size exceeded

### ISSUE THREAD

**Issue by Trollhag** [alvarotrigo/react-fullpage#38]
Hey!

Im having some trouble getting scrollOverflow to work.
If the property is set to true react-fullpage.js throws Maximum call stack size exceeded.

Iv been having inconsequential reproductions in CodeSandbox, But found that npx create-react-app, install react-fullpage and just replace entire index.js with code below reproduces the problem.
Just remove scrollOverflow={true} and it wont throw the error.

```javascript
import React from 'react';
import ReactDOM from 'react-dom';

import 'fullpage.js/vendors/scrolloverflow';
import ReactFullpage from '@fullpage/react-fullpage';

const App = () => (
  <ReactFullpage 
  scrollOverflow={true}
  sectionsColor={['#fff', '#000','#fff', '#000','#fff']}
  render={() => {
      return (
        <div>
            <div className="section">1</div>
            <div className="section"><div style={{height: '150vh'}}>2</div></div>
            <div className="section">3</div>
            <div className="section">4</div>
            <div className="section">5</div>
        </div>
      );
  }}/>
);

ReactDOM.render(<App />, document.getElementById('root'));
```
Am I doing something wrong?

**Comment by alvarotrigo** [comment]
Can you reproduce the issue without using react? Same fullpage.js settings?

**Comment by Trollhag** [comment]
I can not

**Comment by alvarotrigo** [comment]
Hey @Trollhag, I'm not able to replicate your issue on [the example file](https://github.com/alvarotrigo/react-fullpage/blob/master/example/src/index.js) and neither on [this CodeSandBox example](https://codesandbox.io/s/lr0q09q9v9).

Please make sure to use the latest react-fullpage version 0.1.0.

I'll close the issue until an isolated reproduction is provided.

**Comment by Trollhag** [comment]
@alvarotrigo I managed to reproduce the error in [https://codesandbox.io/s/2o1o6x7x6y](CodeSandbox)
I belive its something in fullpage.js/dist/fullpage.extensions.min.js v3.0.3, causing an infinite loop for some reason.

**Comment by alvarotrigo** [comment]
@Trollhag just remove your fullpage.js dependency. It is not necessary.
React-fullpage already includes fullpage.js.

See your updated codesandbox without the fullpage.s dependency:
https://codesandbox.io/s/ym2mrlz9k1

**Comment by Trollhag** [comment]
@alvarotrigo Yes, fullpage.js is set as a dependency to react-fullpage. But it installs v3.0.3, which causes the problem. In CodeSandbox it looks like it uses v3.0.2, which does work, when it isnt forced to v3.0.3 as a dependency? As I said in the first post, creating a new react app and just installing react-fullpage and using scrollOverflow causes the problem.

**Comment by alvarotrigo** [comment]
Thanks for clarifying it!
It is indeed an issue in fullPage.js 3.0.3.

I've fixed it on the dev branch of fullPage.js. Please replace your `vendors/scrollOverflow.js` file for [this one](https://github.com/alvarotrigo/fullPage.js/blob/dev/vendors/scrolloverflow.js) which is the one on the dev branch.

It will get merged in the next release 3.0.4 in about a month.

**Comment by davidciklum** [comment]
Hi @alvarotrigo,
I saw that you've released new version of `react-fullpage` `0.1.3`. 💪
I have a question, have you included a version of `fullPage.js` `3.0.4` that you have mentioned in this issue? Having a look at `package.json` of a new release, there is still the previous version `3.0.3`.

Thanks!
David

**Comment by alvarotrigo** [comment]
@davidciklum I'm above the rules of package.json!! 😈 
fullPage.js 3.0.4 is not yet released, that's why I didn't change it there. However, you'll notice [here](https://github.com/alvarotrigo/react-fullpage/blob/master/dist/react-fullpage.js#L115) how react.fullpage.js is using indeed fullPage.js 3.0.4.

Although, the scrolloverflow vendor file (which is not included by default on react-fullpage.js) need to be manually updated to the latest dev version in fullPage.js. That's why I didn't close this issue yet :)

I'll release 3.0.4 next week. So this issue should be solved by then.

**Comment by alvarotrigo** [comment]
Fixed in the last release of react-fullpage 0.1.9
